src/Symfony/Component/Security/Http/SecurityContextAttributes.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Security\Http;
+
+/**
+ * List of request attributes used along the security flow.
+ *
+ * @author Robin Chalas <robin.chalas@gmail.com>
+ */
+final class SecurityContextAttributes
```

Comment:
Very minor, but what about naming this class `SecurityRequestAttributes` (to avoid any misunderstanding with the "security attributes" used during authorization)

Reply:
Works for me